observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,41 @@
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="monitoring-for-user-defined-projects-first-steps_{context}"]
+== Monitoring for user-defined projects: First steps
+
+As a cluster administrator, you can enable monitoring for user-defined projects in addition to core platform monitoring.
+Doing so enables non-administrator users such as developers to monitor their own projects outside of core platform monitoring.
+
+After you complete the following configuration activities, users can view collected metrics, query these metrics, and receive alerts for their own projects:
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enable user-defined projects.]
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign the `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit` cluster roles] to grant non-administrator users permissions to monitor user-defined projects.
+* If needed, xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-configure-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[assign the `user-workload-monitoring-config-edit` cluster role] to grant non-administrator users permission to configure user-defined projects.
+* xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-alert-routing-for-user-defined-projects[Enable alert routing for user-defined projects] so that developers and other users can configure custom alerts and alert routing for their projects.
+* If needed, configure alert routing for user-defined projects either to xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-the-platform-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects[use the Alertmanager instance deployed for core platform monitoring] or to xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-a-separate-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects[use an optional Alertmanager instance dedicated for use by user-defined projects].
```

Comment:
it the link for `use the Alertmanager instance deployed for core platform monitoring` needed?

Reply:
Good point. Deleted.